source/specs/consensus.rst

```diff
@@ -22,7 +22,7 @@ Staking & Voting
     user wanting to vote must stake their tokens since the voting power is
     weighted by the number of staked tokens, as remarked above.
     
-    All of these requests are performed via a `transaction <./transactions.html#governance-type>`_.
+    All of these requests are performed via a `transaction <./transactions.rst#governance_type>`_.
```

Comment:
It should be `/specs/transactions.html#governance-type-1`
Then there is no need to add new anchor